fiwareskuld/phase/phase0_generateuserlist_alt.py

```diff
@@ -85,7 +85,7 @@ def get_yellow_red_users(self):
                 continue
             if remaining < 0:
                 red_users.append(user)
-            elif remaining <= settings.NOTIFY_BEFORE_EXPIRED:
+            elif remaining <= settings.NOTIFY_BEFORE_TRIAL_EXPIRED:
```

Comment:
there is a different script for community.I didn't want to change too much current script